napari/components/overlays/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .axes import AxesOverlay
```

Comment:
Could overlays be `_overlays` and private? I don't think we think users will import and uses these objects publicaly, so we could hide them which might make iterating and refactoring easier

Reply:
Aren't the equivalents (e.g. `napari.components.axes` public right now though? And they have (and will continue to have?) a pretty prominent/public entry point (i.e. `Viewer.axes` or maybe later `Viewer.overlays.axes` that will need to publicly documented?

Or do you just want to make this module private and import these publicly?

Not that I mind making the public API smaller, but wanted to check.

Reply:
I don't mind either way. I think probably the path of least resistance is to do this change with 0.5 and not deal with all the possible deprecations.